iothub/service/src/IotHubServiceClientOptions.cs

```diff
@@ -59,6 +61,23 @@ public IotHubServiceClientOptions()
         /// </summary>
         public HttpClient HttpClient { get; set; }
 
+        /// <summary>
+        /// Whether to use web sockets or not.
+        /// </summary>
+        /// <remarks>
+        /// Only used for AMQP or MQTT.
```

Comment:
There is no MQTT usage in the service client, so remove it here. On top of that, the user isn't expected to know off-hand which subclients use AMQP, so it may be worth enumerating which ones do here.